### PR TITLE
Bug 1718047 - Reorganize alerts view to show data more compact

### PR DESCRIPTION
**Pull request by esanuandra**

### CODE REVIEW

File: ui/perfherder/alerts/AlertTableRow.jsx [hunk 1]
```diff
@@ -33,6 +32,7 @@ import {
 
 import AlertTablePlatform from './AlertTablePlatform';
 import AlertTableTagsOptions from './AlertTableTagsOptions';
+import AlertTableMagnitude from './AlertTableMagnitude';
```
Comment (beatrice-acasandrei) [inline ui/perfherder/alerts/AlertTableRow.jsx:35]: In the context of Perfherder `AlertTableMagnitude` sounds like this component is responsible for a whole table, but in reality it only handles a cell value from Alert's Table. I think it should have a different name, like `MagnitudeCellValue` / `MagnitudeComponent` or something similar.

**Review by beatrice-acasandrei — APPROVED** [review]

**Comment by esanuandra** [comment]
Before 
image

After
image

**Comment by esanuandra** [comment]
Hey, @sarah-clements ! Could you, please, look over this PR and if everything is alright, merge it? @airimovici is PTO. Thank you!

**Review by sarah-clements — APPROVED** [review]
lgtm

**Pull request merged**